README.md

```diff
@@ -57,16 +74,20 @@ This meta-role requires a permission policy similar to the following:
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
-| account_ids | AWS account IDs that are to be allowed to assume the role | list(string) | [] | no |
-| ssm_names | A list of SSM keys that the created role will be allowed to access. | list(string) | | yes |
-| ssm_regions | AWS regions of target SSMs | list(string) | `["*"]` | no |
-| hostname | The FQDN corresponding to the SSM parameters to be read (e.g. site.example.com) | string | | yes |
+| account_ids | AWS account IDs that are allowed to assume the role. | list(string) | [] | no |
+| iam_username | The username of the IAM user allowed to assume the role.  If not provided, defaults to allowing any user in the specified account(s). | string | `root` | no |
+| role_description | The description to associate with the IAM role (as well as the corresponding policy) that allows read-only access to the specified SSM Parameter Store parameters.  Note that a "%s" in this value will get replaced with the user variable. | string | `Allows read-only access to SSM Parameter Store parameters required for %s.` | no |
+| role_name | The name to assign the IAM role (as well as the corresponding policy) that allows read-only access to the specified SSM Parameter Store parameters.  Note that a "%s" in this value will get replaced with the user variable. | string | `ParameterStoreReadOnly-%s` | no |
+| ssm_names | A list of SSM Parameter Store parameters that the created role will be allowed to access. | list(string) | | yes |
+| ssm_regions | AWS regions of target SSMs (e.g. ["us-east-1", "us-east-2"]).  If not provided, defaults to all regions. | list(string) | `["*"]` | no |
+| user | The name of the entity that the role is being created for (e.g. "test-user" or "host.example.com"). | string | | yes |
```

Comment:
I can see that it might be useful to generalize `hostname`, but do we have any use cases for that yet?  Also, I don't think `user` is the preferred nomenclature.  It's too easy to confuse with IAM users.  How about `entity` instead?

Reply:
`entity-name`

Reply:
I agree that `user` is too close to `iam_username` here.  I'm on board with either `entity` or `entity_name`.  I had considered going that route initially, but wasn't sure if it was any less confusing than `user`.

Our current use cases for this module are:
- The role used by `openvpn-server-tf-module` (hence https://github.com/cisagov/openvpn-server-tf-module/pull/16)
- The `terraform-test-user` code in all of our `*-packer` repos (e.g. [`guacamole-packer`](https://github.com/cisagov/guacamole-packer/blob/develop/terraform-test-user/main.tf#L27-L45)).  In these, we want `entity_name` to be the same as the name of the IAM user being created for test/build purposes.  Note that at the moment, the `terraform-test-user` code does not use this module (it still uses the old [`molecule-packer-ci-iam-user-tf-module`](https://github.com/cisagov/molecule-packer-ci-iam-user-tf-module)), but after this PR is approved, we will be able to do so via the new [`ami-build-iam-user-tf-module`](https://github.com/cisagov/ami-build-iam-user-tf-module).

Reply:
I'm OK with `entity-name`.

Reply:
OK, `entity_name` it is!  Modified in https://github.com/cisagov/ssm-read-role-tf-module/pull/2/commits/8390efb7433d1ad40bfd188466390059b1c1c2c1.